src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -18,6 +18,10 @@
   * `http.host` will no longer be populated on activity. `net.host.name` and
   `net.host.port` attributes will be populated instead.
   ([#3858](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3858))
+  *  Extension method `AddAspNetCoreInstrumentation` on `MeterProviderBuilder` now 
+  supports `AspNetCoreMetricsInstrumentationOptions`. This option class exposes
+   configuration properties fot metric filtering and tag enrichment.
```

Comment:
nits:

* Extra space at the beginning.

* "fot" 🤣

Reply:
fixed